Microsoft.CAP.Portal/Microsoft.CAP.Portal/Microsoft.CAP.Portal.csproj

```diff
@@ -185,6 +185,7 @@
     <Content Include="Scripts\bootstrap.min.js" />
     <None Include="Scripts\jquery-2.2.3.intellisense.js" />
     <Content Include="Scripts\chartHelper.js" />
+    <Content Include="Scripts\clickEventHandler.js" />
```

Comment:
better rename as a more specific name, such as homeButtonClickHandler, because later we may create other Views